Lambda-School-Labs/mydish-fe
# Fix/homepage not updating

Previously, when you created, updated, or deleted a recipe, that change was not reflected in the `HomePage` state. This PR fixes that.

**To test:**
1. Create a new recipe. Save.
2. Check the bottom of the list in the "Home" tab. Is your new recipe there?
3. Edit the title and image of the recipe. Save.
4. Check "Home" again. Have the title and image been updated?
5. Delete the recipe.
6. Return to "Home" one last time. Has the recipe been removed from the bottom of the list?
